src/core/SampleBuffer.cpp

```diff
@@ -287,14 +294,18 @@ void SampleBuffer::update(bool keepSettings)
 				int rate = sfInfo.samplerate;
 				if (frames / rate > sampleLengthMax * 60)
 				{
-					fileLoadError = true;
+					fileLoadError = TooLarge;
 				}
 				sf_close(sndFile);
 			}
+			else if (sndFile == NULL)
```

Comment:
Knowing that libsndfile is written in C, rather than C++, does anyone know if using ``nullptr`` here is safe?

Reply:
Looks like it does:

https://stackoverflow.com/questions/36484473/is-c-null-equal-to-c11-nullptr
https://en.cppreference.com/w/cpp/types/NULL